services/web/server/src/simcore_service_webserver/api_keys/_db.py

```diff
@@ -79,7 +79,7 @@ async def get(
 
             result: ResultProxy = await conn.execute(stmt)
             row: RowProxy | None = await result.fetchone()
-            return ApiKeyInDB.from_orm(row) if row else None
+            return ApiKeyInDB.model_validate(row) if row else None
```

Comment:
so ApiKeyInDB needs to have the from_attribute set to True, so why did you change it on ApiKey model?

Reply:
ApiKey was not changed.